Source/MLX/Documentation.docc/Organization/initialization.md

```diff
@@ -267,3 +267,7 @@ there are specific initializers to request it:
 - ``MLX/repeat(_:count:axis:stream:)``
 - ``MLX/repeat(_:count:stream:)``
 - ``MLX/tri(_:m:k:type:stream:)``
+
+### Copy
+
+- ``MLXArray/copy()``
```

Comment:
What is this? We don't have a copy in MLX python.

Reply:
Ah, something I needed for `compile()` -- it doesn't really copy it, just gives me a new reference to the context (underlying `mlx::core::array`).  I will make this private to the framework.

I think this just got added to the docs because I saw it was listed at the top level.  Not new, but internal use only.